lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
Checking here I found the following:
- cache just avoid the use and creation of cached image but still process it with sharp.
- based on your code comment bellow. The idea of useOriginal is just related to avoid optimizedOriginal step. It would be better to avoid double mean for it.
// ignore optimized original if request explicitly opting out
What we need is to enabled disabling of webp optimization request without operation steps.
So I seems what make more sense is to rename option as bellow:
optimization: false
optimization: { originals: false }
optimization: { originals: { ct: 'image/jpeg' } }